opensearch/src/main/java/org/opensearch/sql/opensearch/response/OpenSearchResponse.java

```diff
@@ -92,23 +112,56 @@ public Iterator<ExprValue> iterator() {
         return (ExprValue) ExprTupleValue.fromExprValueMap(builder.build());
       }).iterator();
     } else {
+      boolean includeId = includes.stream().anyMatch(
+          include -> include.equalsIgnoreCase(METADATA_FIELD_ID)
+      );
```

Comment:
I forgot a bit. Could you double check if  identifier is case sensitive in our engine. Current code means both `_ID` and `_SCORE` are acceptable? If we're case sensitive, that means only lowercase like `_id` is acceptable and probably you can simplify all these with `includes` a hash set instead?

Reply:
The metafields are case-sensitive.  `_ID` will throw a `SemanticCheckException`.   
I will update this to use a map lookup (one already exists).   So I can make this a little cleaner - although I still want to save this to a variable to avoid doing the lookup for each row.

Reply:
updated in ccdc290fa9fcbc40b6beeefe55f6d8e3ab93125b to execute cheks over the includes list